src/implementations/twig/components/card/card.story.js

```diff
@@ -18,123 +9,187 @@ import dataCardTile from '@ecl/specs-component-card/demo/data--tile';
 import card from './card.html.twig';
 import notes from './README.md';
 
-const iconsList = [];
-generalIcons.forEach((icon) => {
-  iconsList.push(icon);
-});
+const getArgTypes = (data) => {
+  const argTypes = {};
+  argTypes.title = {
+    type: { name: 'string', required: true },
+    defaultValue: data.card.title.label,
+    description: 'The card title',
+    table: {
+      type: { summary: 'string' },
+      defaultValue: { summary: '' },
+      category: 'Content',
+    },
+  };
+  argTypes.description = {
+    type: 'string',
+    defaultValue: data.card.description,
+    description: 'The card description',
+    table: {
+      type: { summary: 'string' },
+      defaultValue: { summary: '' },
+      category: 'Content',
+    },
+  };
+  if (data.card.meta) {
+    argTypes.meta = {
+      name: 'meta (comma separated)',
+      type: { name: 'array' },
+      defaultValue: data.card.meta,
+      description: 'The card meta',
+      table: {
+        type: { summary: 'array' },
+        defaultValue: { summary: '[]' },
+        category: 'Content',
+      },
+    };
+  }
+  if (data.card.image) {
+    argTypes.image = {
+      type: 'string',
+      defaultValue: data.card.image.src,
+      description: 'The url of the card image',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '' },
+        category: 'Content',
+      },
+    };
+  }
+  if (data.card.infos) {
+    const infos = data.card.infos.map(({ label }) => label);
+    argTypes.infos = {
+      name: 'infos (comma separated)',
+      type: 'array',
+      defaultValue: infos,
+      description: 'Additional elements like a location or a date',
+      table: {
+        type: {
+          summary: 'array of objects [{ label: "", path: "", icon: {} }]',
```

Comment:
I don't think we need such a complex structure here. The path is not visible for instance, and having a default icon could be enough (that's just a showcase of the component, having one icon or another here won't make a big difference)